### ISSUE TITLE
Collection of issues - 0.7.777

### ISSUE THREAD

**Issue by nekatu** [ldtteam/minecolonies#947]
Hey guys- 

Looking forward to the Bakery & Food system, but having a few issues with the 0.7.777 build. I've gathered that there are hotfixes for the issues; at least all bar the warehouse one. Just wondering when the next version is due that will have these as we're losing resources left right and centre, which is bad enough normally but I've recently moved my server to the "Colonial Expansion" modpack which makes survival (or resource gathering) much harder, so when you lose your stuff it's all the more devastating.

### Minecolonies version

version 0.7.777

### Expected behavior

- Supply Camp can be placed
- Items appear in warehouse chests
- Items from Warehouses arrive at their destination
- Items from Workers appear in their huts

### Actual behaviour

* Items taken to Warehouse disappear before they are placed in chests
* Items taken from Warehouse to huts disappear on handoff
* Items from Workers disappear when placed in the hut inventory
* Supply Camp cannot be placed

### Steps to reproduce the problem

Just allowing the colony to run as normal; workers all appear to go from place to place but no items appear at their destination. 

Plus the supply camp doesn't work at all which has put a few of my players on hold just gathering resources as they are VERY keen to try out the camp.

Anyway- I know you're hard at work on these, so just looking for an ETA on a fixed build.

Keep up the good work- we love this mod! :-)

**Comment by marvin-bitterlich** [comment]
I think they are already fixed and released.

Check the issues for these problems and see if they are closed. If that happens, curse has a builkd containing a fix.

(And please, one Issue per Problem 😋 )

**Comment by nekatu** [comment]
Thanks- for some reason I didn't think to check curse...

Testing the 0.7.832 build now.

**Comment by nekatu** [comment]
From an initial look it seems the issues are still present; do I need to repair the buildings to have the chests replaced?

Also, attempting to place the supply camp results in all minecolonies citizens and workers moving in slow motion. The building tape goes up and the builder stars to clear the ground but doesn't seem to get any further than that.

Will test in a new world this evening though just in case there's something wrong with my current world. I'll gather some logs too.

And just to add to it- all my builders since 0.7.777 seem to carry a bone around with them...

**Comment by Raycoms** [comment]
Can you send us the log?

**Comment by nekatu** [comment]
Will do tomorrow night when I’m back home.

**Comment by nekatu** [comment]
actually- as this is a server game, it should be fine sending you logs from there, right?

Attached are the most recent logs; let me know if there's anything else you need.


[Logs.zip](https://github.com/Minecolonies/minecolonies/files/981695/Logs.zip)

**Comment by nekatu** [comment]
Fresh logs from a quick test from my laptop-

[Server.zip](https://github.com/Minecolonies/minecolonies/files/981751/Server.zip)
[Client.zip](https://github.com/Minecolonies/minecolonies/files/981752/Client.zip)

**Comment by Raycoms** [comment]
Is this 1.10 or 1.11?

**Comment by nekatu** [comment]
1.11.2

**Comment by Raycoms** [comment]
Ahh could you try the alpha version we uploaded to curse if that worked out?

2017-05-07 17:40 GMT-03:00 nekatu <notifications@github.com>:

**Comment by nekatu** [comment]
Testing now- thanks!

**Comment by nekatu** [comment]
[client-alpha-crash.zip](https://github.com/Minecolonies/minecolonies/files/981932/client-alpha-crash.zip)
[server-alpha.zip](https://github.com/Minecolonies/minecolonies/files/981933/server-alpha.zip)
[client-alpha.zip](https://github.com/Minecolonies/minecolonies/files/981931/client-alpha.zip)

Loaded in to a tonne a messages about starving villagers, and then a crash the second I clicked on a worker (the Miner).

Loading back up now, but logs attached FYI.

**Comment by nekatu** [comment]
Same again; interestingly it only crashes the client not the server.

[client-alpha-2.zip](https://github.com/Minecolonies/minecolonies/files/981936/client-alpha-2.zip)
[server-alpha-2.zip](https://github.com/Minecolonies/minecolonies/files/981937/server-alpha-2.zip)

**Comment by nekatu** [comment]
Server restarted when I started building the supply camp-

[server-alpha-3.zip](https://github.com/Minecolonies/minecolonies/files/981945/server-alpha-3.zip)
[client-alpha-3.zip](https://github.com/Minecolonies/minecolonies/files/981946/client-alpha-3.zip)

**Comment by nekatu** [comment]
hmm- server stuck in a reboot loop whenever I log in now; may need to try restore a copy of my world.

any commands to freeze all workers or something like that so I can cancel the build request on that supply camp?

**Comment by nekatu** [comment]
Changed builderBuildBlockDelay to 1000 in config file which allowed the server to run for long enough for me to get to the Town Hall, but like the workers, which I right click on it the client crashes.

[client-alpha-4.zip](https://github.com/Minecolonies/minecolonies/files/981959/client-alpha-4.zip)
[server-alpha-4.zip](https://github.com/Minecolonies/minecolonies/files/981960/server-alpha-4.zip)

Never a dull moment in modding, is there? ;-)

**Comment by Raycoms** [comment]
No, never =)

Currently working on some background systems. As soon as I finished the
part I'll take a look at this.

2017-05-07 18:44 GMT-03:00 nekatu <notifications@github.com>:

**Comment by nekatu** [comment]
Thanks! :-)

really appreciate all the hours you guys put into this; one of the best mods ever.

**Comment by Raycoms** [comment]
Okay I have a fix for the clientside crash. Stupid thingy =D

Okay what's missing is the thingy with the camp

**Comment by Raycoms** [comment]
Okay fixed all of it. I'll post a link to it here

**Comment by Raycoms** [comment]
http://teamcity.smithsmodding.com/repository/download/Minecolonies_PullRequests/878:id/minecolonies-universal-1.11.2-0.7.878.jar

Try out that one

**Comment by nekatu** [comment]
testing now; thanks!

**Comment by Raycoms** [comment]
http://teamcity.smithsmodding.com/repository/download/Minecolonies_PullRequests/881:id/minecolonies-universal-1.11.2-0.7.881.jar
should resolve the dumping problems as well

2017-05-07 23:29 GMT-03:00 nekatu <notifications@github.com>:

**Comment by nekatu** [comment]
Thanks- supply camp worked great in 0.7.878 but was about to say that the dumping issue was still a problem. It looked like items were being shifted by the Warehouse block but again they're just disappearing into the ether; and my server restarted again randomly.

Will try 0.7.881 and then share logs if it happens again.

**Comment by nekatu** [comment]
Server still stuck in a reboot loop as it was when I started the supply camp previously, though there's no building going on now, that I'm aware of.
[server-5.zip](https://github.com/Minecolonies/minecolonies/files/982261/server-5.zip)

**Comment by nekatu** [comment]
Server reboots seem to have stopped after 5 or 6 instances.

Will continue testing.

**Comment by Raycoms** [comment]
Can't find anything related to minecolonies in the logs

2017-05-08 0:11 GMT-03:00 nekatu <notifications@github.com>:

**Comment by nekatu** [comment]
Game seems stable, so that may have been a fluke.

Dumping still seems to be an issue; tried rebuilding my warehouse as a test and even at level 1 all I'm getting is "Warehouse full" messages even though the chests are all empty.

[client-6.zip](https://github.com/Minecolonies/minecolonies/files/982284/client-6.zip)
[server-6.zip](https://github.com/Minecolonies/minecolonies/files/982285/server-6.zip)

**Comment by nekatu** [comment]
Back to reboot cycle every few minutes-

[server-7.zip](https://github.com/Minecolonies/minecolonies/files/982321/server-7.zip)
[server-8.zip](https://github.com/Minecolonies/minecolonies/files/982320/server-8.zip)

Client seems fine though

**Comment by nekatu** [comment]
Stuff is definitely being picked up from the Warehouse chest itself, I've dumped a load of bread in there and the deliveryman is taking it out to the various workers. Will test if the same is true of manually placing items in chests in the warehouse, but it definitely isn't taken stuff from the main warehouse chest and placing it in the other chests in the building as it's convinced the warehouse is full.

**Comment by nekatu** [comment]
Workers are definitely unloading to their chests again, so that's a good sign.

**Comment by nekatu** [comment]
Server reboots definitely seem timed to builder work on the warehouse; restarted again when I tried to upgrade the warehouse.

[server-9.zip](https://github.com/Minecolonies/minecolonies/files/982375/server-9.zip)
[server-10.zip](https://github.com/Minecolonies/minecolonies/files/982377/server-10.zip)

The upgrades do go through but seem to trigger a reboot as soon as they start and the reboot loop continues until building work has completed.

**Comment by nekatu** [comment]
Still having the server boot look on 0.7.887; logs to follow.

**Comment by Raycoms** [comment]
Can you send us the part of the log where it restarts, didn't find any
"minecolonies" related exception in the previous loegs.

2017-05-08 17:52 GMT-03:00 nekatu <notifications@github.com>:

**Comment by nekatu** [comment]
Digging through the logs myself now though but server reboot definitely happens as soon as builder starts work.

I deleted the config file to let it recreate and all was fine as long as it was night time, but the second the sun came up and the builder started work the server restarted.

[Client-11.zip](https://github.com/Minecolonies/minecolonies/files/984955/Client-11.zip)
[Server-11.zip](https://github.com/Minecolonies/minecolonies/files/984954/Server-11.zip)

**Comment by nekatu** [comment]
Testing 0.7.920 to see if it resolves the issue now...

**Comment by nekatu** [comment]
[client-12.zip](https://github.com/Minecolonies/minecolonies/files/987645/client-12.zip)

Crash within minutes of loading; trying again now.

**Comment by nekatu** [comment]
So far so good after that initial crash; building upgrade is happening as it should.

Testing chests now though.

**Comment by Raycoms** [comment]
Can't explain that client window crash for now. Its so strange. Citizen
windows always cause that crash?

2017-05-09 15:34 GMT-03:00 nekatu <notifications@github.com>:

**Comment by nekatu** [comment]
Just tested it again, and yes it does seem to be as soon as I select a citizen.
Haven't had a chance to test the warehouse yet.

[client-13.zip](https://github.com/Minecolonies/minecolonies/files/987717/client-13.zip)

Thinking I may start a new world just to test in case something in one of the previous alphas, or the original warehouse bug I had has broken something.

**Comment by nekatu** [comment]
yep, no sign of any change at my warehouse. Will generate a new world in a bit and test on that.

**Comment by nekatu** [comment]
Loading up 0.8.930 in a test world to see if this solves my issues, and will then test it in my server world.

(Shouldn't it be 0.7.930?)

**Comment by nekatu** [comment]
Selecting a citizen seems OK so far; just waiting on my colony to finish building to test the warehouse.

**Comment by nekatu** [comment]
Items in deliveryman inventory still disappearing at handoff to the Warehouse.

**Comment by Raycoms** [comment]
Only in handoff to the warehouse or also if lumberjack drops of or miner?

2017-05-09 20:29 GMT-03:00 nekatu <notifications@github.com>:

**Comment by nekatu** [comment]
Drop off from all workers to hut chests seems to be working fine.
Pickup from worker hut chests to Deliveryman inventory seems fine.
Drop off from Deliveryman to Warehouse not working.

Interestingly, pickup from items manually placed in the Warehouse hut chest by the Deliveryman seems fine, and drop off from Deliveryman;s inventory to worker hut chests works fine too.

So it's specifically the drop off TO the Warehouse and movement of items FROM the Warehouse hut chest to other chests in the warehouse that is not working.

**Comment by Raycoms** [comment]
Is this 1.10 and 1.11 related or only one of them? do you know?

2017-05-09 21:00 GMT-03:00 nekatu <notifications@github.com>:

**Comment by nekatu** [comment]
Only aware of it in 1.11

**Comment by nekatu** [comment]
Still finding the same bug in 1.11.2-0.7.946 with items from deliveryman's inventory disappearing at Warehouse hut.

Doesn't feel related to other mods, especially as I've not updated or added any other mods, but I may see if there are updated versions available. My mod list is as follows, if that helps any-

- AdChimneys-1.11.2-2.0.9.1-build.0040.jar
- animania-1.11.2-1.0.4.7.jar
- AppleTrees-1.11.2-V2.jar
- AutoSapling 1.11.2-1.3.5.jar
- BedBugs-1.11.2-1.1.7.jar
- Better With Mods-1.1.7-1.11.2.jar
- BetterAchievements-1.11.2-0.4.3.12.jar
- BetterFps-1.4.3.jar
- BloodMagic-1.11-2.1.9-78.jar
- bwncrldd-1.0.jar
- cfm-4.1.2-mc1.11.2.jar
- Chameleon-1.11-3.0.2.jar
- ChestTransporter-1.11.2-2.7.7.jar
- ChunkAnimator-MC1.11-1.2.jar
- ClayBucket-1.11-1.0.jar
- CoffeeSpawner-1.11.2-1.0.1.jar
- ColorUtility-universal-1.0.3.jar
- ColossalChests-1.11.2-1.5.2.jar
- compatlayer-1.11.2-0.2.5.jar
- CookingForBlockheads_1.11.2-5.3.20.jar
- CoralReef-1.4.jar
- CosmeticArmorReworked-1.11.2-v1a.jar
- CraftingCraft_1.11-5.1.1.jar
- CraftingTableIV-1.4.113.jar
- CraftingTweaks_1.11.2-7.1.7.jar
- CreativeCore v1.7.3 mc1.11.2.jar
- CyclopsCore-1.11.2-0.10.2.jar
- D3Core-1.11.2-1.3.0.66.jar
- dailies-1.11.2-6.jar
- DefaultOptions_1.11.2-8.2.2.jar
- Durability Show-1.11.2-4.1.0.jar
- DynamicLights-1.11.2.zip
- easybreading-1.11.2-4.1.0.jar
- ElecCore-1.11-1.7.418.jar
- EmergentVillages-2.2.jar
- enderpay-1.11.2-0.0.3.jar
- ExtraRails-1.11-1.2.1.jar
- Farseek-1.11-2.1.jar
- FasterLadderClimbing-1.11-0.1-118.jar
- FastLeafDecay-v12.jar
- felling-1.0.jar
- ForgeEndertech-1.11.2-2.0.8.0-build.0025.jar
- Forgelin-1.4.2.jar
- furnus-1.11.2-1.93.jar
- GiacomosBookshelf-1.11.2-1.1.jar
- GiacomosFishingNet-1.11.2-1.5.1.jar
- GiacomosFoundry-1.11.2-1.3.12.jar
- GraveStone Mod 1.7.2.jar
- gud_coolers-2.0.0.4.jar
- gud_lib-1.2.0.0.jar
- Guide-API-1.11-2.1.0-51.jar
- hammerz-1.11.2-0.1.4.27.jar
- HoloInventory-1.11.2-2.0.2.143.jar
- hopperducts-mc1.11.2-1.4.9.jar
- HowBoutThatFurniture-1.2.1.jar
- Hwyla-1.8.13-B26_1.11.jar
- InventoryTweaks-1.62+beta.86.jar
- ironchest-1.11.2-7.0.18.800.jar
- ItemPhysic Full 1.4.4 mc1.11.2.jar
- jei_1.11.2-4.3.3.270.jar
- journeymap-1.11.2-5.4.6.jar
- LargeVeins-1.11-1.0.0.jar
- limelib-1.11.2-1.5.4.jar
- literalascension-1.11.2-1.0.1.1.jar
- LittleTiles v1.4.9 mc1.11.2.jar
- llibrary-1.7.4-1.11.2.jar
- Lumberjack-1.11.2-1.3.1.48.jar
- minecolonies-universal-1.11.2-0.7.946.jar
- missing_pieces-1.11.2-4.1.2.jar
- morefuelsmod-1.11.x-1.6.0.jar
- moreoverlays-1.11-mc1.11.2.jar
- MultiMine-1.11.2.jar
- mxtune-1.11.2-0.4.0-dev.23.jar
- NaturesCompass-1.11.2-1.3.0.jar
- NetherPortalFix_1.11-4.3.1.jar
- NoMoreRecipeConflict-0.8(1.11).jar
- notenoughwands-1.1x-1.5.4.jar
- OnlinePicFrame v1.4.0 mc1.11.2.zip
- OptiFine_1.11.2_HD_U_B8_MOD.jar
- Pam's HarvestCraft 1.11.2h.jar
- Plants-1.11.2-beta-0.8.5.jar
- platforms-1.11.0-1.3.4.jar
- Pulver-1.11.x-1.0.0.2.jar
- PvPTime-1.0.12.jar
- railsplus-1.0.1.jar
- redstonepaste-mc1.11.2-1.7.4.jar
- RefinedRelocation-mc1.11.2-3.1.8.jar
- reptilemod-1.11.2-3.6.2-67.jar
- roots-2-0.016.jar
- rustic-0.2.jar
- ShadowMC-1.11.2-3.7.5.jar
- shetiphiancore-1.11.0-3.4.4.jar
- SimpleInventoryStarter-0.2B[1.11].jar
- Stellar API-0.6.4.1.3.jar
- Stellar Sky-0.6.5.1.4.jar
- StorageBoats-2.2.jar
- StorageDrawers-1.11.2-4.2.3.jar
- strait-1.11-0.3.jar
- Streams-1.11-0.3.jar
- StructuredCrafting-1.11.2-0.1.7.jar
- Survivalist-1.11.2-1.9.0.jar
- terraqueous-1.11.0-1.3.10.jar
- Tesla-1.11-1.3.0.51.jar
- ToolBelt-1.11.2-1.6.1.jar
- torohealth-1.11.2-11.jar
- toroquest-1.11.2-3.4.jar
- underphangables-1.11.2-1.0.4.jar
- UnderwaterUtilities-1.11.2-1.2.3.jar
- uniquecrops-1.11.2-0.1.2.jar
- usefulnullifiers-1.3.4.jar
- vending-1.11.2-1.4.3.jar
- villagermarket-0.2-HF.jar
- Waddles-1.11-0.5.1.jar
- WailaHarvestability-mc1.11-1.1.9.jar
- wolfarmor-1.3.3.6.jar
- zz-backpacks 1.11.2 - 3.1.9.jar
- [1.11.2  1.11] Cheese mod v 5.0 by NuclearBanana.jar

Latest log from from the client: 
[latest.zip](https://github.com/Minecolonies/minecolonies/files/991244/latest.zip)

**Comment by nekatu** [comment]
At the risk of jinxing it, as of 1.11.2-0.7.985 this issue seems to be resolved.

Nice work, guys!

:-)
